### ISSUE TITLE
Is it possible to use npm run generate?

### ISSUE THREAD

**Issue by kodekrendel** [sentimony/sentimony-nuxt#1]
Hi there.

Awesome project you have. I am learning Nuxt now, and wonder is it possible to use such setup (Firebase + Nuxt) with static generated files?

PS: офигел, когда понял что это репо чела с Псишайна :smile: 
Я сидел там где-то в 2007 году, причем довольно плотно, хе-хе.

**Comment by ihororlovskyi** [comment]
Yes, its possible, you can simply add new script at `package.json`:

  ```
"scripts": {
    "start": "nuxt",
    "generate": "nuxt generate"
}
```
PS: хаха, прикольно :) 
зацени тогда новую компилу https://sentimony.bandcamp.com/album/va-the-ten :)

**Comment by kodekrendel** [comment]
@ihororlovskyi первый трек шикарен :)

Хм, попробовал я `generate` но заметил, что все равно отправляются запросы к json на Firebase. Зачем, если при generate Axios должен был извлечь все данные? Нельзя ли сделать так, чтоб приложение больше не обращалось к Firebase после `generate`?
